### ISSUE TITLE
OPENAI_API_KEY not set

### ISSUE THREAD

**Issue by adampaigge** [eumemic/gpt-legion#1]
Needed to create a .env file reading

OPENAI_API_KEY = {insert openai api key here}

Can pick up API keys here

https://platform.openai.com/account/api-keys

**Comment by eumemic** [comment]
Yeah, documentation is sparse at the moment, feel free to make a PR to add this to the README!
